README.md

```diff
@@ -83,9 +73,9 @@ The `svcli` utility also provides a small example of using *μthenticode*'s APIs
 You can build it by passing `-DBUILD_SVCLI=1` to `cmake`:
 
 ```bash
-cmake -DBUILD_SVCLI=1 ..
-cmake --build .
-./src/svcli/svcli /path/to/some.exe
+cmake -DBUILD_SVCLI=1 -B build -S . -DCMAKE_TOOLCHAIN_FILE=<vcpkg-path>/scripts/buildsystems/vcpkg.cmake
+cmake --build build
+./build/bin/svcli /path/to/some.exe
```

Comment:
It's a little confusing, but I _think_ this will be:

```suggestion
./build/src/svcli /path/to/some.exe
```

Reply:
Ah right, sorry I saw `svcli` in the `bin` directory because I also installed it (which created the `bin` directory)